PowerEditor/src/NppCommands.cpp

```diff
@@ -4208,6 +4207,8 @@ void Notepad_plus::command(int id)
 			case IDM_SEARCH_CLEAR_BOOKMARKS:
 			case IDM_SEARCH_INVERSEMARKS:
 			case IDM_EDIT_SELECTALL:
+			case IDM_EDIT_INS_TAB:
+			case IDM_EDIT_RMV_TAB:

```

Comment:
OK I can reproduce the duplication.
But why IDM_EDIT_INS_TAB & IDM_EDIT_RMV_TAB can be removed but not always?